Gruntfile.js

```diff
@@ -104,6 +108,26 @@ module.exports = function(grunt) {
       // usage with this name: grunt jshint:files
       files: ['**/*.js']
     },
+
+    uglify: {
+      options: {
+        compress: {
+          global_defs: {
+            'API_KEY': process.env.API_KEY
```

Comment:
Add TURN_URL here too. Since someone cloning this repo would probably have a different url. E.g. something akin to CEOD.